pilot/tools/debug/pilot_cli.go

```diff
@@ -120,7 +120,8 @@ func NewPodInfo(nameOrAppLabel string, kubeconfig string) *PodInfo {
 }
 
 func (p PodInfo) makeNodeID() string {
-	return fmt.Sprintf("sidecar~%s~%s.%s~%s.svc.cluster.local", p.IP, p.Name, p.Namespace, p.Namespace)
+	//return fmt.Sprintf("sidecar~%s~%s.%s~%s.svc.cluster.local", p.IP, p.Name, p.Namespace, p.Namespace)
+	return fmt.Sprintf("ingress~%s~%s.%s~%s.svc.cluster.local", p.IP, p.Name, p.Namespace, p.Namespace)
```

Comment:
This does not look right, we probably need the node type, sidecar or ingress.

Reply:
this is just for my local test :) still WIP, will polish it later